blocks/library/cover-image/index.js

```diff
@@ -185,6 +187,31 @@ registerBlockType( 'core/cover-image', {
 			</section>
 		);
 	},
+
+	deprecated: [ {
+		attributes: attributesDefinition,
```

Comment:
Do you think it'd be reasonable for omitting the `attributes` property from a deprecated definition to default to the current attribute set, since we're not changing anything about it? I guess this could be a concern for future maintainability if we did decide to later change it.

Reply:
The problem we this, is that an undefined value means no attributes right now. I guess we can still use `{}` as a replacement. The same could be said for `supports`. I don't feel strongly either way. Both have drawbacks:

 - The necessity to repeat the value if no changes
 - The necessity to understand that `{}` is different than omitting the property for the second option.